### PR TITLE
move to javascript

### PR DESCRIPTION
**Pull request by nbittich**
OP-1166

### CODE REVIEW

**Comment by nbittich** [comment]
move to https://github.com/lblod/privacy-centric-service/pull/5

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "privacy-centric-service",
+  "version": "1.0.0",
```
Comment (claire-lovisa) [inline package.json]: Version should be 0.5.0 no ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "description": "",
```
Comment (claire-lovisa) [inline package.json]: Maybe a small description + the repo informations ? :)

**Review by claire-lovisa — APPROVED** [review]
Tiny comments, but it's very good :smile:  I tested and it works perfectly from what i could see !

**Pull request merged**